Tasks/distinct.js

```diff
@@ -1,19 +1,16 @@
 // Return an array without duplicates
 
-DISTINCT = (data) => {
-  console.log({ data });
-  A = new Set();
-  w = 0;
-  data.forEach((a) => {
-    if (A.has(a)) {
-      delete data[w];
-    } else {
-      A.add(a);
-    }
-    w++;
-  });
-  return data.filter
-  (x => typeof x === 'number');
+"use strict";
+
+const distinct = (array) => {
+  console.log({ array });
+  const result = [];
+  for (const item of array) {
+    if (!result.includes(item)) {
+      result.push(item);
```

Comment:
I think it would be better to understand `element` instead of `item`
```suggestion
  for (const element of array) {
    if (!result.includes(element)) {
      result.push(element);
```